src/twisted/web/_element.py

```diff
@@ -19,15 +22,9 @@ class Expose:
     Instances of this class can be called with one or more functions as
     positional arguments.  The names of these functions will be added to a list
     on the class object of which they are methods.
-
-    @ivar attributeName: The attribute with which exposed methods will be
-    tracked.
     """
 
-    def __init__(self, doc=None):
-        self.doc = doc
```

Comment:
Is this a public class? That is, is this perhaps a compat issue?

Reply:
`Expose` is in a private module and not re-exported in any public module, so the class itself is private API.